packages/pds/src/auth-routes.ts

```diff
@@ -35,12 +40,34 @@ export const createRouter = ({ oauthProvider, cfg }: AppContext): Router => {
   if (oauthProvider) {
     router.use(
       oauthMiddleware(oauthProvider, {
-        onError: (req, res, err, message) => {
-          oauthLogger.error({ err, req }, message)
+        onError: (req, res, err, msg) => {
+          if (!ignoreError(err)) {
+            oauthLogger.error({ err, req: reqSerializer(req) }, msg)
+          }
         },
       }),
     )
   }
 
   return router
 }
+
+function ignoreError(err: unknown): boolean {
+  if (err instanceof InvalidGrantError) {
+    return (
+      err.error_description === 'Refresh token expired' ||
+      err.error_description === 'Refresh token exceeded inactivity timeout' ||
+      err.error_description === 'Invalid refresh token'
+    )
+  }
```

Comment:
I am less sure about these ones, since this indicates users are getting logged out, right?  That may be interesting to be able to measure.